kde-apps/kdeedu-data/kdeedu_data_kf6-24.12.3.recipe

```diff
@@ -0,0 +1,47 @@
+SUMMARY="Files for educational applications"
+DESCRIPTION="Shared icons, artwork and data files for educational applications"
+HOMEPAGE="https://invent.kde.org/education/kdeedu-data"
+COPYRIGHT="2010-2025 KDE Organisation"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="https://download.kde.org/stable/release-service/$portVersion/src/kdeedu-data-$portVersion.tar.xz"
+CHECKSUM_SHA256="2961d58f7d80e38babe724bb7225369d1619c54d28b3bd05d8dcd333be9d666f"
+SOURCE_DIR="kdeedu-data-$portVersion"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
```

Comment:
@jmairboeck any thoughts here? The package only contains icons and data, so no library or binary that is build.
The Qt6Core requirements is needed (dep for ECM).

Reply:
I would keep it as such. If there are really no secondaryArch runtime dependencies, you could make this "any" and use a hack similar to this https://github.com/haikuports/haikuports/blob/0574de90ae7253236177c7c5b56298ff7bf2853b/dev-perl/extutils_libbuilder/extutils_libbuilder-0.09.recipe#L32-L41 for BUILD_REQUIRES etc., but I don't think it is worth it in this case.

(For TEST_REQUIRES, it is different because that isn't used for the build itself but only locally.)

Reply:
No tests involved, will leave it at that then, thanks.